License added

rupertford opened the pull request:
Added a bsd 3-clause license in its own directory and then added it to all Python files.

rupertford committed:
Create LICENSE

rupertford committed:
issue #11 Added license information to the top of all Python files

rupertford committed:
issue #11 reformatted license in python code as one line was too long

arporter commented on LICENSE:
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, Science and Technology Failities Council
```
Unfortunately you too have got a typo in your license: "Failities" and, just like mine, it's in every file :-(

arporter reviewed (changes requested):
License has been added to all files *apart* from .travis.yml and the 'template' files in the examples. I think the latter is OK but please add it to the travis config file.

Please also correct the typo ("Failities").

rupertford committed:
Issue #11, pull request #12. Fixed typo in license and added license to .travis.yml file

rupertford commented:
Oops, typo has been fixed and license added to the .travis.yml file.

arporter commented:
Can't see any way of formally doing a second review on here but never mind. The changes are good. Confusingly landscape is saying new errors were found but if I click on 'details' and go to the landscape website it says that no errors were introduced. Anyway, this pull request may now be merged.

rupertford commented:
I agree the error is strange. I changed the default values in landscape so it would fail if any new errors were found, but as you say, not new errors are reported. Anyway I've changed the settings to fail if the quality goes down (the default is to pass in any case).
